modules/sc-mesh-secure-deployment/src/nats/scripts/identity.py

```diff
@@ -0,0 +1 @@
+MODULE_IDENTITY="00301a4fc7e4100000001e8cac33"
```

Comment:
I guess that's what it is for now but just in case it helps to mention: needs to be replaced by birth certs as soon as possible :)

Reply:
this file needs to be removed.  It is auto-generated when asking identity.   Identity is needed for communication with NATS servers.  Something similar needs to be on place also after birth certs.

Reply:
fixed in https://github.com/tiiuae/mesh_com/commit/b67d5f9e3c84fcec3732cb04a35297e5acbe08f4

File removed as automatically generated.   Birth cert things were not in the scope.